### PR TITLE
Tutorial harmonic oscillator configuration space

### PR DESCRIPTION
**Pull request by emptymalei**
PR precedent #8 
For #9 

## Added 

- tutorials for HO: 127 0 0 1_8000_hamiltonian_flow_tutorials_harmonic_oscillators_

### CODE REVIEW

**Comment by cmp0xff** [comment]
Why is this a separate PR, instead of being identical to the one of HO?

File: docs/tutorials/harmonic_oscillators.py
[the remark this answers is not in the document]
Comment (emptymalei) [reply docs/tutorials/harmonic_oscillators.py:1]: > A single harmonic oscillator needs to be distinguished with two (coupled) harmonic oscillators... Using plural may not be a good idea. What do you think.

Good point. Let me rename it.

**Comment by emptymalei** [comment]
> Why is this a separate PR, instead of being identical to the one of HO?

The first PR has too many files.

File: .gitignore [hunk 1]
```diff
@@ -158,3 +158,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+.vscode
+docs/tutorials/*.ipynb
```
[the remark this answers is not in the document]
Comment (emptymalei) [reply .gitignore:163]: I thought about the suggestions you made in this comment: https://github.com/kausalflow/hamiltonian_flow/pull/8#pullrequestreview-1899759871

I guess it is better to keep out jupytext. jupytext can be used as a personal tool but it is better we don't enforce it.

Now I switched to the percent format vscode is using for notebooks.

**Pull request merged**